subscribe.go

```diff
@@ -75,16 +79,16 @@ func NewSubscriberPartiallySynced(
 	return l, nil
 }
 
-func newSubscriber(ctx context.Context, dt dt.Manager, topic *pubsub.Topic, onClose func() error, policy PolicyHandler) (*legSubscriber, error) {
-
+func newSubscriber(ctx context.Context, dt dt.Manager, topic *pubsub.Topic, onClose func() error, policy PolicyHandler, selector ipld.Node) (*legSubscriber, error) {
 	ls := &legSubscriber{
-		dt:      dt,
-		topic:   topic,
-		onClose: onClose,
-		updates: make(chan cid.Cid, 5),
-		subs:    make([]chan cid.Cid, 0),
-		cancel:  nil,
-		policy:  policy,
+		dt:              dt,
+		topic:           topic,
+		onClose:         onClose,
+		updates:         make(chan cid.Cid, 5),
```

Comment:
Please specify why a channel of size 5 is used.  Or, change it to either 0 or 1.